client/src/Components/Stars.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
Since this component contains the rating percentages as well as the stars themselves, a better name for the component would be `Rating` or something similar.